src/GovUk.Education.ExploreEducationStatistics.Data.Services/SubjectMetaService.cs

```diff
@@ -335,16 +297,6 @@ private string GetTotalValue(Filter filter)
             return _filterItemRepository.GetTotal(filter)?.Id.ToString() ?? string.Empty;
         }
 
-        private static LabelValue MapLocationAttributeToLabelValue(ILocationAttribute unit)
```

Comment:
Seeing this get removed has just made me think we might be missing something similar in `BuildLocationAttributeViewModel` where we need to `GetCodeOrOldCodeIfEmpty` for local authorities.

Reply:
Good shout, I've fixed this across both services now so that the returned `LocationAttributeViewModel.Value` fields are set to a new `ILocationAttribute.GetCodeOrFallback` method (which gets the old code in the case of local authorities).